Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -0,0 +1,43 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "files"]);
+
+/**
+* Verb to install FAudio
+* see: https://github.com/Kron4ek/FAudio-Builds
+* @returns {Wine} Wine object
+*/
+Wine.prototype.faudio = function () {
+    var setupFile = new Resource()
+        .wizard(this.wizard())
+        .url("https://github.com/Kron4ek/FAudio-Builds/releases/download/19.01/faudio-19.01.tar.xz")
```

Comment:
Can we trust this source?

Reply:
If you want to compile FAudio and provided it yourself later on, you can. I just quickly picked up something from this winetricks issue [https://github.com/Winetricks/winetricks/issues/1162](https://github.com/Winetricks/winetricks/issues/1162) to have a proof of concept and a way to test it for Space Engineers, then I ended up in the two problems I listed.